lib/evmone/baseline.cpp

```diff
@@ -193,17 +211,22 @@ evmc_result execute(const VM& vm, ExecutionState& state, const CodeAnalysis& ana
     const auto& cost_table = get_baseline_cost_table(state.rev);
 
     const auto* const code = state.code.data();
-    auto code_it = code;  // Code iterator for the interpreter loop.
-    while (true)          // Guaranteed to terminate because padded code ends with STOP.
+    const auto stack_bottom = state.stack.top_item;
```

Comment:
Why is this not
```suggestion
    const auto stack_top = state.stack.top_item;
```

Reply:
It's kind of confusing that now we there exist `state.stack.top_item` and `position.stack_top` and only one of them is valid.

I guess `ExecutionState::Stack` should be removed soon and replaced with just a pair of stack storage and "initial_stack_top" pointer.

Reply:
This is because Baseline leaves `.top_item` untouched but Advanced is manipulating it (as Baseline did before).
We could more Stack to AdvancedExecutionState, but this would allocate it _after_ the stack storage. We can also migrate Advanced to `StackTop`. All to be decided later and with low priority.

So far I used `state.stack.storage` here instead.